code/modules/mob/dead/observer/observer.dm

```diff
@@ -150,6 +150,7 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	. = ..()
 
 	grant_all_languages()
+	setup_hud_traits()
```

Comment:
Does this actually work? The proc is  already called by `. = ..()` if I don't err here, and I don't see why calling it twice would do the trick, but that'd also mean something wrong is going on deeper down.

Reply:
idk why it does the trick, but it works on my machine. Maybe because mob/dead never calls `. = ..()`

Reply:
You're right!